ChangeLog.d/fix_return_type_for_invalid_crypto_key.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fixes the issue #4162. Return PSA_ERROR_INVALID_HANDLE instead of
```

Comment:
The changelog entry still does not follow the style of other changelog entries and is not grammatically correct.

* “fixes <s>the</s> issue #4162”, and we normally write “Fixes #nnnn.” at the end.
* “Return<s>s</s>”
* “if **an** invalid key is passed”
* For other compliance issue, we've been using the language “bring X in line with version 1.0.0 of the [PSA Crypto API] specification.”

See my suggested text above: https://github.com/ARMmbed/mbedtls/pull/4198#discussion_r596251675

Reply:
Hello Giles.  I have updated the changelog entry. It fulfills all the requirements mentioned in the section "How to update Changelog Entry" on https://github.com/ARMmbed/mbedtls/blob/development/ChangeLog.d/00README.md. However, since you have some specific requirements on wordings, I would suggest if you can update the check_files.py to include these additional checks as well! :)

Reply:
Your update still doesn't address all my comments.

AI is not advanced enough to automatically enforce grammar and style, let alone relevance and completeness.

Reply:
Hello Giles, I think my latest Bugfix statement is grammatically correct, and clearly establishes the context. May I know what needs updating? Thanks